tests/requirements.txt

```diff
@@ -2,4 +2,6 @@ pysys == 1.6.1
 requests == 2.25.1
 toml == 0.10.2
 retry == 0.9.2
+junitparser == 2.2.0
+junit2html == 30.0.6
```

Comment:
These changes are unrelated to this fix, right? Accidental commit from your other work trying to generate Junit style test reporting for pysys?

Reply:
That's a separate fix. We need both of the packages to analyse records.